swap/src/protocol/alice.rs

```diff
@@ -133,10 +135,10 @@ impl From<message3::OutEvent> for OutEvent {
 pub struct Behaviour {
     pt: PeerTracker,
     amounts: Amounts,
-    message0: Message0,
-    message1: Message1,
-    message2: Message2,
-    message3: Message3,
+    message0: Message0Behaviour,
+    message1: Message1Behaviour,
+    message2: Message2Behaviour,
+    message3: Message3Behaviour,
```

Comment:
```suggestion
    message0: message0::Behaviour,
    message1: message1::Behaviour,
    message2: message2::Behaviour,
    message3: message3::Behaviour,
```

I believe it is better to use the module to qualify a type than having (redundant because right now it is `message0::Message0Behaviour`) qualifier in the name.